lib/auth/auth.go

```diff
@@ -5261,6 +5265,8 @@ func (a *Server) Ping(ctx context.Context) (proto.PingResponse, error) {
 		features.IdentityGovernance = a.accessMonitoringEnabled
 	}
 
+	features.AccessGraph = a.accessGraphEnabled
```

Comment:
I would vote for reusing the pattern for `Plugins`:

Can't say I particularly like this, as this will result in confusingly desynced `Features()` in Auth server itself vs what is returned to the proxy.

I would suggest reusing this pattern:

https://github.com/gravitational/teleport.e/blob/e5403beda326e90e8562c8a73831087ffb673573/tool/modules/modules.go#L111-L139

and then call `EnableAccessGraph()` from `auth/plugin.go` in `e` (this will mesh well with the "is team plan" check I'm adding).

Adding a new method `EnableAccessGraph()` makes the merge a bit complicated, since we need to add it in `e` first, and only then in OSS (so that the interface is satisfied). We could do a type-cast i.e. `modules.GetModules().(*enterpriseModules)` for now in `e` and add a `TODO`, I will fix it my PR.

Reply:
Changed